source/_components/scene.tuya.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+title: "Tuya Scene"
+description: "Instructions on how to setup the Tuya scene within Home Assistant."
+date: 2018-07-20 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tuya.png
+ha_category: Scene
+ha_release: 0.75
+---
+
+
+The `tuya` scene platform allows you to activate your [Tuya Smart](https://www.tuya.com) scenes.
+
+The platform supports scenes.
+
+The devices state in panel will not change immediately after you activate scene.
```

Comment:
The device state in the frontend panel will not change immediately after you activate a scene.